Sources/FoundationEssentials/AttributedString/AttributedString+Guts.swift

```diff
@@ -30,7 +30,7 @@ extension AttributedString {
         typealias _AttributeStorage = AttributedString._AttributeStorage
 
         var version: Version
-        var string: BigString
+        package var string: BigString
```

Comment:
It looks like this is only used from one place: `NumberFormatStyleTests` which could easily do `String(self.characters)` instead of `String(self._guts.string)` which would save us from needing to mark this and the imports above as `package`.

For cases like this, would you rather we update them in a separate change and keep this as just a `@testable` --> `package` change, or would you want to incorporate that into this to reduce the number of things that we import as `package` like the collections modules above?

Reply:
I’d prefer if this is just a mechanical `@testable` -> `package` change. It’s fairly big already and I don’t want to hide any other functionality change in it.